src/components/WebexInMeeting/WebexInMeeting.css

```diff
@@ -0,0 +1,22 @@
+.in-meeting {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .remote-media {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .local-media {
+  position: absolute;
+  bottom: 25%;
+  right: 3%;
```

Comment:
How did you arrive at this percentages?

Reply:
no idea. just looks good. any better approach? im trying to leave some room for the controls and other streams at the bottom.

Reply:
We might have to do some more testing in different screen sizes